src/components/DirectorListItem.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import { ListItem, Box } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+
+import PropTypes from 'prop-types'
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    ' & a': { width: '100%' }
+  }
+}))
+
+const DirectorListItem = ({ directorData }) => {
+  const classes = useStyles()
+  const [selectedIndex, setSelectedIndex] = React.useState(1)
+
+  const handleListItemClick = (event, index) => {
```

Comment:
I am a little bit confused with index here, why it's always zero and why is it initialized in the state with 1.

Reply:
it should be for future features, something like drag and drop, etc. It change it's value in onClick fn. Ok, I'm deleting it now.)